### PR TITLE
refactoring

### PR DESCRIPTION
**Pull request by xoxys**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,19 +1,21 @@
-Copyright (c) 2018 Patrick Jahns <pjahns@owncloud.com>
+MIT License
```
Comment (tboerger) [inline LICENSE]: Please drop that, GitHub is displaying it automatically as MIT.

File: Makefile [hunk 2]
```diff
@@ -1,6 +1,7 @@
 IMAGE_NAME ?= owncloudci/core:latest
```
Comment (tboerger) [inline Makefile]: Just drop the Makefile? This is anyway not used from anybody manually?

File: README.md [hunk 3]
```diff
@@ -1,46 +1,46 @@
...
 - `EXCLUDE`
 Exclude files/folders from being copied to the workspace. This is useful when testing apps that are included in the nightly tarballs. The exclude pattern uses rsync `--exclude` logic
 
-
 If `CORE_PATH` is not defined - the plugin assumes the workspace directory is nested two directories deeper in relation to the owncloud folder
 
 With the following definition of the workspace in the `drone.yml`
-```
+
+```Yaml
```
Comment (tboerger) [inline README.md]: These language switches are normally written downcase.

File: README.md [hunk 4]
```diff
@@ -1,46 +1,46 @@
...
 workspace:
    base: /owncloud
    path: apps/app_to_test
 ```
 
 The directory layout would be like this:
-```
+
+```Shell
```
Comment (tboerger) [inline README.md]: This should be `console`

File: README.md [hunk 5]
```diff
@@ -1,46 +1,46 @@
...
-  └── apps  
+  └── apps
      └── app_to_test  ( workspace )
 ```
 
-
 ### Full list of variables
 
-```
+```Shell
```
Comment (tboerger) [inline README.md]: This should be `console`

File: README.md [hunk 6]
```diff
@@ -60,14 +60,16 @@ DB_PASSWORD        (owncloud)
...
+
 ## Examples
 
-**Basic Example**  
+**Basic Example**
 
 Download `owncloud-daily-master-qa.tar.bz2` and put contents into `/var/www/owncloud`, install owncloud with sqlite
-```
+
+```Yaml
```
Comment (tboerger) [inline README.md]: These language switches are normally written downcase.

File: README.md [hunk 7]
```diff
@@ -79,11 +81,11 @@ pipeline:
     version: daily-master-qa
 ```
 
-**Git core branch and mysql as database**  
+**Git core branch and mysql as database**
 
 Fetch stable10 branch into `/drone`, install owncloud with mysql
 
-```
+```Yaml
```
Comment (tboerger) [inline README.md]: These language switches are normally written downcase.

File: README.md [hunk 8]
```diff
@@ -111,7 +113,7 @@ services:
 
 **Exclude folders/files from core**
 
-```
+```Yaml
```
Comment (tboerger) [inline README.md]: These language switches are normally written downcase.

File: Makefile (hunk 2 above)
Comment (xoxys) [reply Makefile]: Don't know. Some repos had it some not. I decided to keep/add it for all

File: README.md (hunk 3 above)
Comment (xoxys) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (xoxys) [reply README.md]: done

File: README.md (hunk 5 above)
Comment (xoxys) [reply README.md]: done

File: README.md (hunk 6 above)
Comment (xoxys) [reply README.md]: done

File: README.md (hunk 7 above)
Comment (xoxys) [reply README.md]: done

File: README.md (hunk 8 above)
Comment (xoxys) [reply README.md]: done

**Pull request merged**